third_party/gpus/rocm_configure.bzl

```diff
@@ -621,6 +634,7 @@ def _create_local_rocm_repository(repository_ctx):
                 repository_ctx,
                 rocm_config.amdgpu_targets,
             ),
+            "%{rocm_gpu_architectures}": str(rocm_config.amdgpu_targets),
```

Comment:
You need a corresponding entry in create_dummy_repository. Otherwise it will not work in cuda builds.
I had the reverse problem when working on cuda support to prevent it failing on rocm.

Reply:
you are right!

have added the corresponding entry in the create_dummy_repository and pushed it out. thanks.